src/modules/groups/application/use-cases/execute-group-shutdown.use-case.ts

```
@@ -0,0 +1,125 @@
+  constructor(
+    private readonly groupRepository: GroupRepository,
+    @InjectRepository(Vm)
+    private readonly vmRepository: Repository<Vm>,
+    @InjectRepository(Server)
+    private readonly serverRepository: Repository<Server>,
+  ) {}
+
+  async execute(
+    groupId: string,
+    dto: GroupShutdownDto,
+    userId: string,
```

ESLint

[error] 'userId' is defined but never used. Allowed unused args must match /^_/u.

(@typescript-eslint/no-unused-vars)